skeleton/aws/variables.tf

```
@@ -0,0 +1,3 @@
+variable "terraform_organization" {}
+variable "terraform_workspace" {}
+variable "region" {}
```

please give some description for the variables
```suggestion
variable "terraform_organization" {
  description = "Organization name on Terraform Cloud"
}

variable "terraform_workspace" {
  default = "Workspace name on Terraform cloud
}

variable "region" {
  default = "AWS region"
}
```